lib/open_mic.rb

```diff
@@ -0,0 +1,28 @@
+class OpenMic
+	attr_reader :location, :date, :performers
+	
+	def initialize(hash)
```

Comment:
we typically don't name variables as their data type (i.e., hash) -- so instead of `hash`, we might call this `open_mic_info`

Great job catching that this was a hash and handling it correctly